README.md

```diff
@@ -2,24 +2,11 @@
 
 > Simple git-hook integration for your gulp workflows.
 
-guppy streamlines and extends your git-hooks by integrating them with your 
-[gulp](http://gulpjs.com) workflow. This enables you to have **gulp tasks that 
-run when triggered by a git-hook**, which means you can do cool things like 
-abort a commit if your tests are failing. Git-hooks can now be managed through 
```

Comment:
Rather than remove the reference, let's just reword it. Using the guppy-hook packages is still a valid choice.